src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/write/direct_es/ElasticSearchWriteAccessDirect.java

```diff
@@ -97,7 +98,6 @@ public class ElasticSearchWriteAccessDirect extends ElasticSearchWriteAccess {
     public ElasticSearchWriteAccessDirect() throws URISyntaxException {
         ElasticSearchConfiguration config = ElasticSearchGlobalConfiguration.get().getElasticSearch();
         if(config != null) {
-            config.readResolve();
```

Comment:
Why is the line removed?
It initializes a lot of properties not just splitMessagesLongerThan.
Is function readResolve() used somewhere else? If not, it should be deleted.

Reply:
Actually it was initially not called. [I changed it here](https://github.com/SAP/elasticsearch-logs-plugin/pull/87/files#diff-4d0e15af1f1c1a7a36265c7c8695d8f864f146918c92a9a13c86d57d0a29c346) to be called. But I guess readObject() and readResolve() methods are such methods [which are being called automatically during serialization](https://www.baeldung.com/java-serialization-readobject-vs-readresolve) right? That is why I removed them as part of this PR.

Reply:
I have learned something new. 
It looks like readResolve() is called internally during serialization.